src/assets/locales/en/staking.json

```
@@ -15,8 +15,12 @@
   "pending rewards": "Pending rewards",
   "restaking": "Restaking",
   "rewards": "Rewards",
-  "your tokens has been restaked successfully": "Your {{ tokens }} have been restaked successfully",
-  "your tokens hasn't been restaked successfully": "Your {{ tokens }} have been restaked unsuccessfully",
-  "rewards claimed successfully": "You rewards have been claimed successfully",
-  "rewards claimed unsuccessfully": "You rewards have been claimed unsuccessfully"
+  "unbond from": "Unbond from",
+  "expected delivery": "Expected delivery",
+  "your tokens has been restaked successfully": "Your {{ tokens }} has been restaked successfully",
+  "your tokens hasn't been restaked successfully": "Your {{ tokens }} has been restaked unsuccessfully",
+  "rewards claimed successfully": "You rewards has been claimed successfully",
+  "rewards claimed unsuccessfully": "You rewards has been claimed unsuccessfully",
+  "your tokens has been unbonded successfully": "Your {{ tokens }} has been unbonded successfully",
+  "your tokens hasn't been unbonded successfully": "Your {{ tokens }} has been unbonded unsuccessfully"
```

All of these should be fixed
```suggestion
  "your tokens has been restaked successfully": "Your {{ tokens }} have been restaked successfully",
  "your tokens hasn't been restaked successfully": "Your {{ tokens }} have not been restaked",
  "rewards claimed successfully": "You rewards have been claimed successfully",
  "rewards claimed unsuccessfully": "You rewards have not been claimed",
  "your tokens has been unbonded successfully": "Your {{ tokens }} have been unbonded successfully",
  "your tokens hasn't been unbonded successfully": "Your {{ tokens }} have not been unbonded"
```